docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
     ports:
       - "80:80"
     volumes:
+      - "./nginx/log:/var/log/nginx:rw,delegated"
```

Comment:
I think you can leave the default behavior (ie `rw` or empty since it's the default value).
But it could be great to add `rw,delegated` on the line below. 😉